test/utils/image/manifest.go

```diff
@@ -250,6 +252,7 @@ func initImageConfigs() map[int]Config {
 	configs[VolumeISCSIServer] = Config{e2eRegistry, "volume/iscsi", "2.0"}
 	configs[VolumeGlusterServer] = Config{e2eRegistry, "volume/gluster", "1.0"}
 	configs[VolumeRBDServer] = Config{e2eRegistry, "volume/rbd", "1.0.1"}
+	configs[HugePageTester] = Config{"docker.io", "odinuge/hugepage-tester", "1.0"}
```

Comment:
This has to be changed to `Config{e2eRegistry, "hugepage-tester", "1.0"}`, and the image has to be built and pushed to the e2eRegistry before merging